snooty.toml

```diff
@@ -19,7 +19,7 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver = "java"
 driver-long = "MongoDB Java Driver"
 driver-short = "Java driver"
-version = "4.11"
+version = "5.0"
```

Comment:
Issue:
I think the "full-version" source constant below indicates that the release is 5.0.1 rather than 5.0.0.

Suggestion:
Update "full-version" to "{+version+}.0"